run-tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eu
+
+lambda_function_name=$(aws cloudformation describe-stacks --stack-name ipv-cri-check-hmrc-smoke-tests --query "Stacks[0].Outputs[?OutputKey=='CanaryInvokerFunction'].OutputValue" --output text)
+canary_names=("nino-canaries")
+
+for canary_name in "${canary_names[@]}"; do
```

Comment:
This is sequential, a future improvement would be to allow the canaries to work in parallel - that'd be a lot easier using TS instead of Bash